src/EditModel/event-program/notifications/NotificationDialog.js

```diff
@@ -71,23 +76,33 @@ const steps = [
         key: 'what',
         name: 'what_to_send',
         content: compose(
-            connect(mapStateToProps, _mapDispatchToProps, undefined, { pure: false }),
+            connect(mapStateToProps, mapDispatchToProps, undefined, { pure: false }),
             createFieldConfigsFor(
                 'programNotificationTemplate',
                 ['name', 'messageTemplate'],
             ),
         )(class extends Component {
             componentDidMount() {
-                // FIXME: Hack to reposition the dialog when the first step is clicked (https://github.com/callemall/material-ui/issues/5793)
-                setTimeout(() => window.dispatchEvent(new Event('resize')), 600);
+                // Hack to reposition the dialog when the first step is clicked (Material-UI bug)
```

Comment:
Should we leave the issue number in the code?

Reply:
Did you check the link? They basically say they won't fix it ;)